### ISSUE TITLE
Fix Configuration.xml

### ISSUE THREAD

**Issue by ronie-z-wave** [Z-Wave-Me/ExpertUI#225]
Maybe related to #216 

I've played a bit around with that issue and figured the following out:

- PUT requests 501 from server (not recognized/known) - is OK i think
- POST/GET will response 404 if conf/Configuration.xml doesn't exist 
  as  I know this file should be added by E-UI right? But untill POST isn't able to add this file it isn't possible in that way ...
- If this file exists (added it manually) GET is working well but POST with changed xml won't change the current one. Request is 200 OK with the unchanged Configuration.xml

checked it with postman:

![configuration-xml_post_404](https://cloud.githubusercontent.com/assets/10545480/22499227/c4eda434-e85d-11e6-93bc-99b3cf3842f3.png)

![configuration-xml_bug](https://cloud.githubusercontent.com/assets/10545480/22499171/6d808388-e85d-11e6-8914-fb9f27fa21b9.png)

**Comment by PoltoS** [comment]
@zwave-mke I propose to replace this Configuration.xml by a special call to ZWaveAPI (tbd in ZWave/index.js) that will save the file in storage as "ExpertUI-Configuration-<zwayname>" (with zway name!!!). Another call will be used to retrive it from the storage.

It will be JSON and not Configuration.xml.

Agree?

**Comment by vades** [comment]
I agree wit that. Some parts of the UI (configuration, assoc ..) will need to be refactored but JSON is better and faster as XML.

**Comment by Kosta-Github** [comment]
so, is there a fix available now?

**Comment by michaelhq** [comment]
solution available?

**Comment by vades** [comment]
There is nothing new... :-(
